### PR TITLE
Fix for issue#48

### PR DESCRIPTION
**Pull request by mistermax80**
Message box for updates in MacOs Version #48

### CODE REVIEW

File: src/main/java/tool/updater/UpdateController.java [hunk 1]
```diff
@@ -104,14 +105,22 @@ public static void check4update(boolean bInformationOnly) {
...
-			return "https://github.com/akasolace/HO/releases/download/dev/HO_" + full_version + ".zip";
+			return "https://github.com/akasolace/HO/releases/download/dev/HO_" + full_version + suffix;
 		} else {
 			String ver = Double.toString(version);
-			return "https://github.com/akasolace/HO/releases/download/" + ver + "/HO_" + ver + ".zip";
+			return "https://github.com/akasolace/HO/releases/download/" + ver + "/HO_" + ver + suffix;
 		}
 	}
 
+	final public static boolean isMac() {
```
Comment (akasolace) [inline src/main/java/tool/updater/UpdateController.java]: a isMac function already exists, could you please use it instead of redefining a new one

**Comment by mistermax80** [comment]
Ok I fully agree with you... I Can I Will do change this night...
Bye thanks you so much!!!!

**Comment by mistermax80** [comment]
@akasolace 
I did all exept:

>  include the piece of code to rename from .zip to OSX.zip inside the if(bInformationOnly) block which will be after the changes directly if(isMac), so you wont need to redefine a isMac function

In get_HOzip_updateURL method I used the static method HOMainFrame.isMac() so I can call that method without redefine the method.
I prefered to do this, beacuse in the get_HOzip_updateURL(String full_version, double version, String versionType) version is double and I cannot passing _OSX string.
Let me know if for you is ok.
If ok, I prefer you test the code on different platform respect to macos.

Thanks.

**Comment by akasolace** [comment]
> I did all exept:
> 
> > include the piece of code to rename from .zip to OSX.zip inside the if(bInformationOnly) block which will be after the changes directly if(isMac), so you wont need to redefine a isMac function
> 
> In get_HOzip_updateURL method I used the static method HOMainFrame.isMac() so I can call that method without redefine the method.
> I prefered to do this, beacuse in the get_HOzip_updateURL(String full_version, double version, String versionType) version is double and I cannot passing _OSX string.
> Let me know if for you is ok.

I am sorry to be a pain, but I would prefer if the method stays clean, I really don't like exception. Hence I see 2 solutions:

1. As mentioned previously you do the name conversion within the isMac block.  Something like
`version = get_HOzip_updateURL.replace(".zip", "_OSX.zip")`

2. or if you prefer you create a new method ``get_HO_OSXpkg_download_url``

Also it would be cleaner to rename frm ``get_HOzip_updateURL`` to ``get_HO_zip_download_url``

**Comment by mistermax80** [comment]
@akasolace , ok no problem, I will do.

**Comment by mistermax80** [comment]
@akasolace 
I guess to did all, as you required 😄 
Let me know if is all ok for you.

**Comment by akasolace** [comment]
I just proposed an updated function description. 
The rest looks good to me.
Thank you !

**Comment by mistermax80** [comment]
@akasolace 
With 7e57552 commit I corrected an introduced bug, that is verified in not macos version (I tried the PR on linux).
Sorry for this inconvenient.

**Comment by akasolace** [comment]
@mistermax80  I was expecting to have only change regarding the function description as my text proposal above but I see you made other changes but you have not change the text. Once you are done, and you want me to merge your PR, let me know explicitly. Thank you !

**Comment by mistermax80** [comment]
@akasolace
Sorry for this misunderstanding... but I had not received your replaced text when I committed the last correction.
I hope really now It's all ok.
I really sorry for all this.

**Comment by akasolace** [comment]
Perfect 
thanks a lot !

**Comment by mistermax80** [comment]
Thank you too!!!

**Pull request merged**